### ISSUE TITLE
select profile when download

### ISSUE THREAD

**Issue by hiddify** [zzzgydi/clash-verge#342]
Would you please select profile after adding a new profile?

**Comment by zzzgydi** [comment]
If you don't have any profiles before, it will select the latest added profile

**Comment by hiddify** [comment]
Yes, however, it would be great if similar to clashforwindows it automatically select the new added url

**Comment by hiddify** [comment]
Or it can also show a dialog that do you want to select it?

**Comment by hiddify** [comment]
https://github.com/zzzgydi/clash-verge/pull/345
